.github/workflows/release-digital-ocean.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: Release Digital Ocean 1-Click
+
+on:
+  release:
+    types: [published]
+
+  push:
+    paths:
+      - "DigitalOceanMarketplace/**"
+
```

Comment:
I wanted to have this as CI for the Packer files and scripts for any future changes. @bitwarden/dept-devops are we ok with the 15min runtime?

Reply:
I don't think we should have the `push` event type at all.  The only time we need to build an image is upon release of the new self hosted scripts.

Reply:
How should we handle validation and integration around the files then? Especially if engineers outside the @bitwarden/dept-devops team makes changes? How will they validate those changes?

Reply:
Building on change to any files in the directory makes sense.  There are two options I've thought of.  My main concern is the amount of image artifacts we'll be making in our Digital Ocean account.

1. We build the image with a static name so it's always overridden.  This allows us to keep one image artifact in there and if we need another past version we can build it again.
2. We create a workflow/job that cleans up image artifacts and leaves X latest images.  The downside to this is the amount of time coding the solution and having another thing to maintain and make sure is working properly.

What do you think?

Reply:
I like both of those ideas. I tested with a static image name, but unfortunately DO assigned a GUID to it so even with a static name, we get duplicate names.

I ended up leveraging packer's manifest output which then I grab the ID after a run and just remove the image with `doctl` if the branch isn't `["rc", "hotfix", "master"]`. This should allow for collaboration from anyone to integrate their changes with the Packer builder.

[See this commit for the changes.](https://github.com/bitwarden/self-host/pull/22/commits/7fee6ca74319c1cf2a8464df7ea0873f74b8864b)

Reply:
I think that'll be good enough for now.  If we run into issues we can put in a fix.